src/tray/templates/services/nginx/configs/marsha.conf.j2

```diff
@@ -99,6 +99,11 @@ server {
     try_files $uri @proxy_to_marsha_xapi;
   }
 
+  location /api/v1/runners {
```

Comment:
Should we use a variable for the api version, or do it later if version changes ?

Reply:
IMO this should not be managed with a variable